openml/extensions/sklearn/extension.py

```diff
@@ -934,12 +955,55 @@ def check(param_grid, restricted_parameter_name, legal_values):
                       '{GridSearchCV, RandomizedSearchCV}. '
                       'Should implement param check. ')
 
-            if not check(param_distributions, 'n_jobs', None):
+            if not SklearnExtension._check_parameter_value_recursive(param_distributions, 'n_jobs', None):
                 raise PyOpenMLError('openml-python should not be used to '
                                     'optimize the n_jobs parameter.')
 
+    def _can_measure_cputime(self, model: Any) -> bool:
+        """
```

Comment:
Maybe we should document that `n_jobs` may not be optimized, and also that time measurements can only be performed with `n_jobs != -1`? The latter might incentivize users to explicitly define their `n_jobs` for multicore runs.

Reply:
> Maybe we should document that n_jobs may not be optimized

Agreed. 

> and also that time measurements can only be performed with n_jobs != -1? 

Agreed that it should be documented. Will do so. For the review, please note that
* cpu time can be measured with `n_jobs = 1` only
* wall time can be measured with `n_jobs` >= 0